Tasks/ServiceFabricUpdateManifests/task.json

```diff
@@ -17,7 +17,7 @@
     ],
     "version": {
         "Major": 2,
-        "Minor": 0,
+        "Minor": 1,
         "Patch": 1
```

Comment:
FYI, you should have set patch to 0 when incrementing the minor version (not a big deal), but for testing you will need to increment it to publish a new version to your instance.